README.md

```diff
@@ -1,4 +1,4 @@
-# meza v0.4
+# meza v0.9-dev
```

Comment:
Seemed like a good idea since this was such a long-running release. Didn't want the REL0_9 to seem like it was the stable released branch. It should be changed prior to merge.